zoo/src/main/scala/com/intel/analytics/zoo/pipeline/inference/InferenceModel.scala

```
@@ -157,8 +188,8 @@ class InferenceModel(private var supportedConcurrentNum: Int = 1,
    * @param weightPath the path of openvino ir bin file
    */
   def doLoadOpenVINO(modelPath: String, weightPath: String): Unit = {
-    if (supportedConcurrentNum > 1) {
-      InferenceSupportive.logger.warn(s"supportedConcurrentNum is $supportedConcurrentNum > 1, " +
+    if (concurrentNum > 1) {
+      InferenceSupportive.logger.warn(s"concurrentNum is $concurrentNum > 1, " +
```

should we throw an exception here? or rephrase the warning message that concurrent prediction would consume lots of memory as  openvino does not support shared weights model copies

---

it will not clone models when it is an openvinomodel, as the warning message said: not supported.

---

ok. should we fail fast here as the input parameter is invalid?

---

I think warning is enough here, it can work to support concurrent requests.